ROADMAP.md

```diff
@@ -1,53 +1,72 @@
+  - [🚑 Self-healing networks](#-self-healing-networks)
+  - [📮 Offline message queue / postbox](#-offline-message-queue--postbox)
+- [Evolve](#evolve)
+  - [✈️ WebTransport](#️-webtransport)
+  - [⏱ Full Observability](#-full-observability)
+  - [🧪 Automated compatibility testing](#-automated-compatibility-testing)
+  - [Stream Migration Protocol](#stream-migration-protocol)
+  - [WebRTC](#webrtc)
+  - [🤖 libp2p as a Wasm library](#-libp2p-as-a-wasm-library)
+  - [🤝 Low latency, efficient connection handshake](#-low-latency-efficient-connection-handshake)
+  - [🛣️ Peer Routing Records](#️-peer-routing-records)
+  - [🗣️ Polite peering](#️-polite-peering)
+  - [🧱 Composable routing](#-composable-routing)
+  - [💣 Attack resistance, threat models and security](#-attack-resistance-threat-models-and-security)
+  - [📈 Proving we are scalable and interoperable](#-proving-we-are-scalable-and-interoperable)
+  - [🌡️ Telemetry protocol](#️-telemetry-protocol)
+  - [📩 Message-oriented transports](#-message-oriented-transports)
+  - [☎️ Reducing the dial fail rate](#️-reducing-the-dial-fail-rate)
+  - [🔀 Peer exchange protocol](#-peer-exchange-protocol)
+  - [🏹 RPC and other common node communication patterns](#-rpc-and-other-common-node-communication-patterns)
+- [Done](#done)
+  - [🕸 Hole punching on TCP and QUIC](#-hole-punching-on-tcp-and-quic)
+
+## Core Tenets
+Before we dive into what libp2p should support and enable, let's outline the core tenets that underpin the project. As maintainers, we commit to ensuring libp2p is:
+- 🔒 Secure - Building on libp2p *improves* an application's security posture. We minimize vulnerabilities and act rapidly to address and report issues.
+- 🪨 Stable - Releases do not unknowingly break deployed features. We maintain cross-version compatibility across libp2p releases and implementations. Deprecations are well-communicated.
+- 🥽 Specified - We thoroughly specify the libp2p framework and its suite of supported protocols, independent of language or implementation. Through specifications, we coordinate and drive future developments.
+- 🚀 Performant - We ensure comparable performance to widely used protocols on the Internet. We avoid performance degradations in new releases.
+
+These tenets form the foundation for features and initiatives listed below.  As a result, we generally prioritize filling any gaps in these tenets above other work.
+
+### Implementation Roadmaps
+
+This roadmap doesn't provide a timeline for project adoption across libp2p implementations. For specific sequencing and dates, see the different implementation roadmaps:
+- [go-libp2p](https://github.com/libp2p/go-libp2p/blob/master/ROADMAP.md)
+- [js-libp2p](https://github.com/libp2p/js-libp2p/blob/master/ROADMAP.md)
+- [rust-libp2p](https://github.com/libp2p/rust-libp2p/blob/master/ROADMAP.md)
+- [libp2p/test-plans](https://github.com/libp2p/test-plans/blob/master/ROADMAP.md)
```

Comment:
minor put I would put this at the top of the list since it's shared across all implementations.  I want this area to be top-of-mind for all :)

Reply:
makes sense, will do 👍